OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -327,9 +338,9 @@ Action InitAudioPanel(Widget panel)
 
 			if (!ss.Mute)
 			{
-				panel.Get<SliderWidget>("SOUND_VOLUME").OnChange += x => Game.Sound.SoundVolume = x;
-				panel.Get<SliderWidget>("MUSIC_VOLUME").OnChange += x => Game.Sound.MusicVolume = x;
-				panel.Get<SliderWidget>("VIDEO_VOLUME").OnChange += x => Game.Sound.VideoVolume = x;
+				panel.Get<ExponentialSliderWidget>("SOUND_VOLUME").OnChange += x => Game.Sound.SoundVolume = x;
```

Comment:
Likewise with all these: you're not actually using the subclass properties, so it is cleaner (not to mention a much smaller diff) to access these via the parent slider class.